yapf/yapflib/py3compat.py

```diff
@@ -116,3 +117,12 @@ class ConfigParser(configparser.ConfigParser):
 
     def read_file(self, fp, source=None):
       self.readfp(fp, filename=source)
+
+def removeBOM(source):
```

Comment:
Please run `yapf` over the code first. (The lines should be two-space indented.)